.pre-commit-config.yaml

```diff
@@ -8,21 +8,24 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
+default_language_version:
+    python: python3
+
 # Styles the code properly
 # Exclude the UI files, as they are auto-generated.
 exclude: "ui\/.*py$"
+
 # List of super useful formatters.
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
+    rev: v5.0.0
     hooks:
     # Ensures the code is syntaxically correct
     - id: check-ast
-      language_version: python3
     # Ensures a file name will resolve on all platform
     - id: check-case-conflict
     # Checks files with the execute bit set have shebangs
-    - id: check-executables-have-shebangs
+    # - id: check-executables-have-shebangs
```

Comment:
What is the problem with this hook @jpbrault ?

Reply:
Good catch. Nothing.

I've copied the file from `tk-desktop` as is as I though @carlos-villavicencio-adsk suggested in https://github.com/shotgunsoftware/tk-flame-export/pull/58 but I realize now that the `tk-desktop` has some test disabled.